contracts/TwabRewards.sol

```diff
@@ -255,10 +260,7 @@ contract TwabRewards is ITwabRewards {
         uint256 _promotionEndTimestamp = _promotion.startTimestamp +
             (_promotion.epochDuration * _promotion.numberOfEpochs);
 
-        require(
-            _promotionEndTimestamp >= block.timestamp,
-            "TwabRewards/promotion-inactive"
-        );
+        require(_promotionEndTimestamp >= block.timestamp, "TwabRewards/promotion-inactive");
```

Comment:
This should just be >....because the promotionEndTimestamp is really the start time of the next epoch.

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/31/commits/d8c980345708961a2d73ec44411deae699f22aca